creator-node/src/services/stateMachineManager/stateMonitoring/index.js

```diff
@@ -41,7 +41,11 @@ class StateMonitoringManager {
         this.processFindReplicaSetUpdatesJob.bind(this)
     })
 
-    await this.startQueue(queue, discoveryNodeEndpoint)
+    // Don't start this queue on local dev because jobs process quickly with 0 users, which spams the logs
+    if (!config.get('creatorNodeIsDebug')) {
```

Comment:
huh interesting that we have this config along with `devMode` ([link](https://github.com/AudiusProject/audius-protocol/blob/master/creator-node/src/index.ts))

this works tho